test/unit/offset.js

```diff
@@ -548,7 +548,7 @@ QUnit.test( "fractions (see #7730 and #7885)", function( assert ) {
 
 	result = div.offset();
 
-	assert.equal( result.top, expected.top, "Check top" );
+	assert.equal( Math.abs( result.top - expected.top ) < 0.25, true, "Check top within 0.25 of expected" );
```

Comment:
Also, maybe we can make this `assert.ok( expression )` instead of `.assert.equal( expression, true )`.